source/codegen/metadata/nifake/functions.py

```diff
@@ -1262,6 +1262,29 @@
         'returns': 'ViStatus',
         'use_session_lock': False
     },
+    'InitExtCal': {
```

Comment:
Since you are already submitting the fake metadata and test for fake metadata in DMM branch in other PR, you don't need to replicate that here. Revert changes to this file, nifake/CHANGES.md and ni_fake_service_tests.cpp files.

Reply:
Done